Formula/spek.rb

```diff
@@ -0,0 +1,25 @@
+class Spek < Formula
+  desc "Acoustic spectrum analyser"
+  homepage "https://www.spek.cc"
+  url "https://github.com/alexkay/spek/releases/download/v0.8.5/spek-0.8.5.tar.xz"
+  sha256 "1bccf85a14a01af8f2f30476cbad004e8bf6031f500e562bbe5bbd1e5eb16c59"
+  license "GPL-3.0-only"
+
+  depends_on "pkg-config" => :build
+
+  depends_on "ffmpeg"
+  depends_on "wxwidgets"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--libdir=#{lib}",
+                          "--disable-silent-rules"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "Spek version #{version}", shell_output("#{bin}/spek --version")
```

Comment:
Thanks for your PR. We need a test that exercises the some of the functionality of the software. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the Formula cookbook.

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation/source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
This software is a GUI program.
The command "spek input.mp3" does not output any file.
The only command-line functions are "--help" and "--version".
How should I do the test?

Reply:
This is probably fine, then.